pkg/route/validation/validation.go

```diff
@@ -354,6 +303,58 @@ func validateTLS(route *routev1.Route, fldPath *field.Path) field.ErrorList {
 	return result
 }
 
+// validateTLSExternalCertificate
+func validateTLSExternalCertificate(ctx context.Context, route *routev1.Route, fldPath *field.Path, sarCreator routecommon.SubjectAccessReviewCreator, secretsGetter corev1client.SecretsGetter) field.ErrorList {
+	tls := route.Spec.TLS
+	var errs field.ErrorList
+
+	errs = append(errs, routecommon.CheckRouteCustomHostSAR(ctx, fldPath, sarCreator)...)
+
+	if err := authorizationutil.Authorize(sarCreator, &user.DefaultInfo{Name: "system:serviceaccount:openshift-ingress:router"},
```

Comment:
Nit pic: the router service account can be a constant.

Reply:
Yeah, agree! Done

Reply:
I have an impression that you may have forgotten to push some commits because I see resolved comments which are not actually addressed.

Reply:
Can you please try accessing https://github.com/openshift/library-go/pull/1625/commits/bb12c499ed386faa3527a37a6ca09a64313c7579 and let me know?